CLAUDE.md

```diff
@@ -0,0 +1,23 @@
+# CLAUDE.md
```

Comment:
question: should we include or not?

Reply:
Feels like yet another thing to manage IMO, but if you don't think it'll need to be modified often then sure, why not I guess?

Reply:
no